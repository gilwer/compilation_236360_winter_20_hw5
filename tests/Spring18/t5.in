//negatives and overflow
void main() {
	printi(0-5);
	
	printi(0b-1);
	
	printi(0b-5b);
	
	printi(2147483647 + 2);
	
	printi(214748365 * 10b);
	
	printi(255b + 2b);
	
	printi(128b * 2b);
	
}