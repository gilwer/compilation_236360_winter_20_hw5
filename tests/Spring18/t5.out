-5
-1
251
-2147483647
-2147483646
1
0
